### ISSUE TITLE
Support for relative file paths

### ISSUE THREAD

**Issue by tresf** [qzind/qz-print#137]
Investigate the possibility of supporting relative file paths in 2.0

``` javascript
var printData = [
   {
      type: 'image',
      data: '/relative/path/to/img/image_sample.png'
// HERE ------^
   }
];
```

Currently, data must be a full path requiring all of our code examples to use a `getPath()` function which our clients won't necessarily have.

**Comment by tresf** [comment]
@bberenz I found a painfully simply way to resolve URLs... It constructs an `<a href="..."/>` tag with the specified URL and reads the value right back out.  It works with IE7 and higher.

I've also placed in a small portion of browser-detection, not sure if it's warranted.

Question... how can we assume when a URL is being supplied to the data element?  Can we determine that in js, or is the logic all Java-side?

``` javascript
function makeAbsolute(url) {
    if (document && typeof (document.createElement == 'function')) {
        var a = document.createElement('a');
        a.href = url;
        return a.href;
    }
    return url;
}
```
